<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-041P56D" ignored="2" name="Default test(failed)" tests="6" failures="1" timestamp="14 Oct 2021 07:39:06 GMT" time="0.147" errors="0">
  <testcase name="test3" time="0.046" classname="org.deepan.Reruntest">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.016" classname="org.deepan.Reruntest">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.0" classname="org.deepan.Reruntest">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.0" classname="org.deepan.Reruntest">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.0" classname="org.deepan.Reruntest">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.0" classname="org.deepan.Reruntest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.deepan.Reruntest.test3(Reruntest.java:18)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test" time="0.0" classname="org.deepan.Reruntest">
    <ignored/>
  </testcase> <!-- test -->
  <testcase name="test2" time="0.0" classname="org.deepan.Reruntest">
    <ignored/>
  </testcase> <!-- test2 -->
</testsuite> <!-- Default test(failed) -->
